### PR TITLE
Improve Vagrant Development Environment

### LINKED ISSUE #1520: Improve Vagrant Development Environment
- Update development environment dependencies
Dependencies should include @virtualization package group

- Docs are not compiling via ansible task

Compilation error
```
Problem in http domain: field is supposed to use role 'obj', but that role is not in the domain
```
-  Exec Paths of librariesio_consumer.py & check_service.py services in systemd service files  are not correct

### PR DESCRIPTION
**Pull request by nikitabugrovsky**
Closes #1520

### CODE REVIEW

File: news/1402.bug [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix Python versioning parse doesn't return string
```
Comment (Zlopez) [inline news/1402.bug:1]: This file shouldn't be here.

File: ansible/roles/anitya-dev/files/librariesio-consumer.service [hunk 2]
```diff
@@ -4,7 +4,7 @@ After=network.target
 
 [Service]
 Environment="ANITYA_WEB_CONFIG=/home/vagrant/anitya.toml"
-ExecStart=librariesio_consumer.py
+ExecStart=/home/vagrant/devel/anitya/librariesio_consumer.py
```
Comment (Zlopez) [inline ansible/roles/anitya-dev/files/librariesio-consumer.service:7]: You shouldn't need that the script should be in standard PATH, it is installed as binary

File: ansible/roles/anitya-dev/files/check-service.service [hunk 3]
```diff
@@ -5,7 +5,7 @@ Documentation=https://github.com/fedora-infra/anitya/
 
 [Service]
 Environment="ANITYA_WEB_CONFIG=/home/vagrant/anitya.toml"
-ExecStart=check_service.py
+ExecStart=/home/vagrant/devel/anitya/check_service.py
```
Comment (Zlopez) [inline ansible/roles/anitya-dev/files/check-service.service:8]: You shouldn't need that the script should be in standard PATH, it is installed as binary

**Review by Zlopez — COMMENTED** [review]
I'm not sure why `aniyta/check_service.py` and `anitya/librariesio_consumer.py` are even in this PR and I also added few other comments

File: ansible/roles/anitya-dev/files/librariesio-consumer.service (hunk 2 above)
Comment (nikitabugrovsky) [reply ansible/roles/anitya-dev/files/librariesio-consumer.service]: yes, you are right, changed path to scrip, provided by binary install

File: ansible/roles/anitya-dev/files/check-service.service (hunk 3 above)
Comment (nikitabugrovsky) [reply ansible/roles/anitya-dev/files/check-service.service]: yes, you are right, changed path to the script, provided by binary install

**Comment by nikitabugrovsky** [comment]
@Zlopez could you kindly take another look?

**Comment by nikitabugrovsky** [comment]
Zlopez done!

**Comment by nikitabugrovsky** [comment]
Looks like I did not pick up upstream changes before rebasing. Pushed again.

**Comment by Zlopez** [comment]
It's still not correct, I can see 2 commits in the https://github.com/fedora-infra/anitya/pull/1521/commits, the news file is one of them.

**Comment by Zlopez** [comment]
It's possible that you created this branch from different branch than master?

**Comment by nikitabugrovsky** [comment]
I guess I just need to recreate PR... Locally I o

**Pull request closed without merging**